Source/WebCore/SourcesCocoa.txt

```diff
@@ -291,7 +291,7 @@ platform/audio/ios/MediaSessionHelperIOS.mm @nonARC @no-unify
 platform/audio/ios/MediaSessionManagerIOS.mm @nonARC @no-unify
 platform/audio/mac/AudioBusMac.mm @nonARC
 platform/audio/mac/AudioOutputUnitAdaptorMac.cpp
-platform/audio/mac/AudioHardwareListenerMac.cpp
+platform/audio/mac/AudioHardwareListenerMac.cpp @no-unify
```

Comment:
Always annoying to have more of these. Can we avoid this?

Reply:
I have moved this enum declaration to `CoreAudioSPI.h` for when the internal SDK is not used because it comes from a private header otherwise.